shared/core/src/firebase/observable-db-ref.ts

```diff
@@ -59,6 +57,9 @@ export class ObservableDbRef {
   }
 
   on(eventName: string): Observable<firebase.database.DataSnapshot> {
-    return fromEventPattern(handler => this.ref.on(eventName, handler), handler => this.ref.off(eventName, handler));
+    return fromEventPattern(
+      handler => this.ref.on(eventName, (val: firebase.database.DataSnapshot) => handler(val)),
+      handler => this.ref.off(eventName, handler)
```

Comment:
Should we invoke the `handler` with a lambda here as well? Just to be consistent with the other one?

Reply:
Yes, we can totally do that. Something's off with Travis anyways, so I'll prolly have to rebase again either way.